saleor/checkout/fetch.py

```diff
@@ -63,7 +63,7 @@ def fetch_checkout_lines(checkout: "Checkout") -> Iterable[CheckoutLineInfo]:
         "variant__product__collections",
         "variant__channel_listings__channel",
         "variant__product__product_type",
-    )
+    ).select_related()
```

Comment:
This drops 12 queries from the test case. I think this might be due to Django autocaching intermediary lookups that weren't listed (lines 70 and 71)